directory_forms_api_client/actions.py

```diff
@@ -47,8 +47,13 @@ def save(self, data, form_session=None,):
 class SaveOnlyInDatabaseAction(AbstractAction):
     name = 'save-only-in-db'
 
-    def __init__(self, *args, **kwargs):
-        self.meta = {}
+    def __init__(self, subject, full_name, email_address, *args, **kwargs):
```

Comment:
As this is changing signature it would be major update right ?
Also can we not get from **kwargs

second thought is are these all mandatory ?

Reply:
This never worked before and only used for a single form that has this bug. They kind of mandatory fields which view to pass ing to directory-forms-api-client to identify who's submitting this request.

Reply:
ok I still suggest upgrading the major number as there's other projects using this code base. So if by small chance they are using this method they will know it's an interface change.

Reply:
ok, Amended version to `6.0.0`

Reply:
@depsiatwal  if you are happy then please approve this

Reply:
👍